app/src/main/res/values/strings.xml

```diff
@@ -537,14 +537,14 @@
 
 
   <!--APPC-->
-    <string name="appc_title_iab">Use Your AppCoins</string>
+  <string name="appc_title_iab">Spend Your AppCoins</string>
```

Comment:
Were these changes approved by Carlos? Wasn't he supposed to be the one changing the text?